license.go

```diff
@@ -237,7 +237,7 @@ forUnknownFiles:
 		if len(licenses) == 0 {
 			parts := strings.Split(name, `/`)
 			for i := len(parts) - 1; i > 0; i-- {
-				for _, licName := range []string{`LICENSE`, `LICENCE`, `LICENSE.md`, `LICENCE.md`, `LICENSE.txt`, `LICENCE.txt`} {
+				for _, licName := range []string{`LICENSE`, `LICENCE`, `LICENSE.md`, `LICENCE.md`, `LICENSE.txt`, `LICENCE.txt`, `COPYING`, `COPYING.md`, `COPYING.txt`} {
```

Comment:
As long as there needs to be a whitelist, can `LICENSE.MIT` also be included? [go-errors/errors](/go-errors/errors) uses it.

Reply:
That's a good observation, but I can't get all the possible variations, unfortunately. The LICENSE, LICENCE, and COPYING variations are all very standard places for license information that applies to a whole directory. You can use a .dependency_license file to explicitly document the license for anything that isn't automatically detectable.